src/urlValidation/urlValidation.js

```diff
@@ -0,0 +1,31 @@
+function urlValidation(url) {
+  const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+  const [textToConvert, searchParams] = url.split('?');
+
+  const errors = [];
+  const params = new URLSearchParams(searchParams);
+  const toCase = params.get('toCase');
+  const normilizedText = textToConvert.slice(1);
+
+  if (textToConvert.length < 2) {
+    errors.push({
+      message: `Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".`,
```

Comment:
You can create constants for all app messages.

Reply:
@DanilWeda are you good with back end?